torch/testing/_internal/common_nn.py

```diff
@@ -98,6 +98,15 @@ def get_weight(m):
 # - `has_parity`: if `False`, expects this test dict to fail the C++ parity test. Default: True.
 
 module_tests = [
+    dict(
```

Comment:
Note that these dicts are on the way out and will eventually be replaced with `ModuleInfo` entries. Happy to chat more about this if you're interested - they are a module analogue to `OpInfo`, which I'm sure Mike covered in a previous lesson.